test/integration/http-test.js

```diff
@@ -608,6 +608,73 @@ describe('HTTP APIs', function () {
         })
     })
 
+    describe('content-type-based file extensions', () => {
+      // ensure the container exists
+      before(() =>
+        server.post('/post-tests/')
+          .send(postRequest1Body)
```

Comment:
`postRequest1Body`??

Reply:
I know… just reusing what the other tests do. The whole test file needs refactoring.